### DISCUSSION TITLE
Can i do anything with version 107.0.5304.94

### DISCUSSION THREAD

**Discussion by winghf** [3kh0/ext-remover#270] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Can i do anything with it like remove extensions or like a way to make it go to a lower version

**Comment by winghf** [comment, marked as answer]
I have crosh unblocked used by some magic is there a way to downgrade using crotch

**Reply by frogmasamune** [reply]
yes, check TN discord (discord.gg/unblock), will probably be blocked on chromebook so do it on personal computer. That's where most exploit progress is made

**Reply by md4r2** [reply]
its risky to downgrade so u might as well just wait for [Pingot](https://github.com/Holding-THE-CREATOR/Pingot)

**Reply by bartholemuew** [reply]
pingot dead

**Comment by Aka-but-nice** [comment]
Yea there is in crosh type "rollback" doesn't work for me cuz my school disabled the rollback command but, who knows might for you!

**Reply by Alex2Mayhem** [reply]
> Yea there is in crosh type "rollback" doesn't work for me cuz my school disabled the rollback command but, who knows might for you!

That works for me, but it says it'll powerwash your device to the previous version*** (i did not hit Y so i didn't really do it) i don't suggest this though *edit is does not work*

**Reply by Alex2Mayhem** [reply]
gotta pull out a spare chromebook to test it out

**Reply by Aka-but-nice** [reply]
no, it does work for me but when I press y it says like error and it is gonna powerwash, so I don't recommend doing it cuz the school may find out
